hangman/interface.py

```diff
@@ -43,32 +49,127 @@ def write_instructions(self):
             "3. If the letter you guessed doesn't belong to the word, an appendage to the stickman hangman will be drawn and you will lose one unit of life."
         )
         print(
-            "4. You have 9 tries (lives) to guess the word. You lose the game when you run out of lives and the character is hung.\n\n"
+            "4. You have 9 tries (lives) to guess the word. You lose the game when you run out of lives and the character is hung."
+        )
+        self.clear()
+        print(
```

Comment:
nitpick, but can we have everything on the same line?

Reply:
> nitpick, but can we have everything on the same line?

Do you mean like this:
`print("4. You have 9 tries (lives) to guess the word. You lose the game when you run out of lives and the character is hung.")`

Reply:
yup exactly!

Reply:
I blame the `black `formatter for this